backend/src/main/java/gov/cdc/usds/simplereport/db/repository/TestEventRepository.java

```diff
@@ -6,15 +6,20 @@
 import gov.cdc.usds.simplereport.db.model.Organization;
 import gov.cdc.usds.simplereport.db.model.Person;
 import gov.cdc.usds.simplereport.db.model.TestEvent;
+import org.springframework.data.jpa.repository.Query;
 
 public interface TestEventRepository extends AuditedEntityRepository<TestEvent> {
-
 	public List<TestEvent> findAllByPatient(Person p);
 
 	public List<TestEvent> findAllByOrganizationOrderByCreatedAtDesc(Organization o);
 
 	public List<TestEvent> findAllByOrganizationAndFacility(Organization o, Facility f);
 
-    public TestEvent findFirst1ByPatientOrderByCreatedAtDesc(Person p);
+	public TestEvent findFirst1ByPatientOrderByCreatedAtDesc(Person p);
 
+	// todo: simple_report.simple_report.test_event should be replaced
+	@Query(value="SELECT * FROM simple_report.simple_report.test_event q WHERE q.created_at > (cast(?1 as timestamp)) " +
```

Comment:
As mentioned in other discussions, we can use the persistence layer and configuration constants to solve this better.

Reply:
Yeah, I'll need to get help from a Spring expert to help me craft the right set of java and object and spring keywords to replace this line of sql.

Reply:
resolving so we can do a springify second pass.